src/Components/ProductTile.js

```diff
@@ -12,24 +21,100 @@ import { useHistory } from 'react-router';
 //          - NA
 const ProductTile = ({ products, index, setProducts }) => {
   const history = useHistory();
-  const upVote = () => {
-    const updatedProduct = { ...products[index] };
-    let currentVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
-    updatedProduct.upVoted = !updatedProduct.upVoted;
-    updatedProduct.downVoted = false;
-    let newVote = updatedProduct.upVoted ? 1 : (updatedProduct.downVoted ? -1 : 0);
-    updatedProduct.votes = updatedProduct.votes - currentVote + newVote;
-    setProducts(products.map((product) => product.id === products[index].id ? updatedProduct : product));
+
+  const username = ReactSession.get("username");
+  const [loggedin, setLoggedin] = useState(username !== '');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8496218)
```suggestion
  const [loggedin, setLoggedin] = useState(username !== "");
```